OptionSelector/AnimalDetailTableViewController.h

```diff
@@ -0,0 +1,29 @@
+//
+//  AnimalDetailTableViewController.h
+//  OptionSelector
+//
+//  Created by Lauren Caponong on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CQCategory.h"
+
+@class AnimalDetailTableViewController;
+
+@protocol ViewControllerAnimalDetailTableViewDelegate <NSObject>
+- (void)addItemViewController:(AnimalDetailTableViewController *)controller didFinishEnteringItem:(CQCategory *)item;
+@end
+
+
+
+
+@interface AnimalDetailTableViewController : UITableViewController
+
+@property (nonatomic) CQCategory *thing;
```

Comment:
It's best to use variable names that are as specific as possible. Usually names like `number` or `myArray` or `thing` aren't that useful, because they don't provide much context about _why_ that variable exists and what it does.